examples/vite-react-with-typescript/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vite-react-with-typescript",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "media-chrome": "link:../../",
+    "react": "link:../../node_modules/react",
```

Comment:
See comment for the equivalent declaration in the next.js example app's package json for an explanation of this.